opts/quotedstring.go

```diff
@@ -22,6 +22,10 @@ func (s *QuotedString) String() string {
 }
 
 func trimQuotes(value string) string {
+	if len(value) < 2 {
+		return value
+	}
+
 	lastIndex := len(value) - 1
 	for _, char := range []byte{'\'', '"'} {
 		if value[0] == char && value[lastIndex] == char {
```

Comment:
Not really important, but I was looking at the `for .. range` loop here, which felt a bit over-complicated, and could just be checking for both variants (`'` , `"`);

```go
func trimQuotes(value string) string {
	if len(value) < 2 {
		return value
	}
	if (value[0] == '\'' || value[0] == '"') && value[len(value)-1] == value[0] {
		return value[1 : len(value)-1]
	}
	return value
}
```

Doing a quick bench shows that it's at least more performance (0.3ns vs 5.1ns, so ~ 15x faster)

```
BenchmarkTrimQuotes-8       	228469384	         5.175 ns/op	       0 B/op	       0 allocs/op
BenchmarkTrimQuotes_new-8   	1000000000	         0.3302 ns/op	       0 B/op	       0 allocs/op
```


No need to change in this PR (it's a really tiny optimisation, and not in a critical code-path), but perhaps I'll do a follow-up after this.